packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -312,7 +312,10 @@ const INDIVIDUAL_ACTIONS = [
     level: 'primary',
   },
   {
-    handler: (remote, { editRemote }) => editRemote(remote),
+    handler: (remote, { reset, editRemote }) => {
```

Comment:
```js
handler: (remote, { reset, editRemote }) => editRemote(remote).then(reset)
```

Reply:
`editRemote` only changes the state and isn't async. So the name is [definitely confusing](https://github.com/vatesfr/xen-orchestra/pull/4707#discussion_r358222116)! :)
The reset after the actual edition is correctly done here:
https://github.com/vatesfr/xen-orchestra/blob/80af1fd5f0cf78f978534f655391b512fbc01660/packages/xo-web/src/xo-app/settings/remotes/remote.js#L56-L68